src/server/cmd/worker/main.go

```diff
@@ -56,9 +56,12 @@ type appEnv struct {
 
 func main() {
 	// Copy the contents of /pach-bin/certs into /etc/ssl/certs
+	copyErr := false
```

Comment:
Why don't you just return the error from `Walk` and then log it below. I think that would condense this code a lot, you'd only need 1 line to handle each error rather than 3 and you wouldn't need `copyErr`.

Reply:
If walk encounters an error, it stops walking, and my logic was that if there's some reason we can't copy one cert we should still try to copy the others. Let me know what you think

Reply:
Oh right, makes complete sense. :+1: